utilities/virt.py

```diff
@@ -1630,7 +1631,7 @@ def assert_vm_not_error_status(vm: VirtualMachineForTests, timeout: int = TIMEOU
 
 def wait_for_running_vm(
     vm: VirtualMachineForTests,
-    wait_until_running_timeout: int = TIMEOUT_4MIN,
+    wait_until_running_timeout: int = TIMEOUT_8MIN,
```

Comment:
why do we need to x2 all default timeout values here?

Reply:
we need to understand why you hit a timeout; such a change is not acceptable as it adds potentially  4 minutes for each test that calls this function.